_includes/updates/2020-10-28.html

```diff
@@ -0,0 +1,11 @@
+<div class="ds-u-padding-y--2">
```

Comment:
Can we add an additional update example to see how the page looks with multiple entries?

Reply:
@mtrang1263 added another update to the page